### DISCUSSION TITLE
The differences between Neovim packages on Archlinux/Aur

### DISCUSSION THREAD

**Discussion by vollowx** [ayamir/nvimdots#816] [Q&A] [answered]
I'm just looking for Neovim newly package (nightly/git), and I found 3 packages.

- `neovim-git`
- `neovim-nightly`
- `neovim-nightly-bin`

What's the difference?/Which should I use?

**Comment by ayamir** [comment, marked as answer]
> I'm just looking for Neovim newly package (nightly/git), and I found 3 packages.
> 
> * `neovim-git`
> * `neovim-nightly`
> * `neovim-nightly-bin`
> 
> What's the difference?/Which should I use?

Use `neovim-git`
